apps/www/app/routes/_dashboard.chunks.tsx

```diff
@@ -1,7 +1,87 @@
+import {cn} from '~/lib/utils';
+import {getChunksByProjectId} from '~/models/chunk.server';
+import {requireProjectId} from '~/session.server';
+
+export const loader = async ({request}: LoaderFunctionArgs) => {
+  const projectId = await requireProjectId(request);
+  const chunks = await getChunksByProjectId(projectId);
+
+  return json({
+    projectId,
+    chunks,
+  });
+};
+
 export default function Chunks() {
+  const {projectId, chunks} = useLoaderData<typeof loader>();
+  const {chunkId} = useParams();
+
   return (
-    <div className="flex justify-center items-center">
-      <h1 className="font-mono text-3xl text-muted">/chunks</h1>
-    </div>
+    <motion.div
+      initial={{opacity: 0}}
+      animate={{opacity: 1}}
+      transition={{duration: 1}}>
+      <ResizablePanelGroup direction="horizontal" className="h-full w-full">
+        <ResizablePanel defaultSize={30} minSize={30} maxSize={70}>
+          <div className="border-b px-4 space-y-0.5 h-[75px] flex flex-col justify-end pb-1">
+            <h2 className="text-2xl font-bold tracking-tight">Chunks</h2>
+            <p className="text-xs text-muted-foreground">
+              Project <span className="text-xs font-mono">{projectId}</span>
+            </p>
+          </div>
+          <ScrollArea className="h-[calc(100vh-140px)] px-4">
+            {!chunks.length && (
+              <p className="text text-muted-foreground">No chunks published</p>
+            )}
+            {chunks.length &&
+              chunks.map(it => (
+                <Link
+                  to={`/chunks/${it.id}`}
+                  key={it.id}
+                  className={cn(
+                    'flex flex-col items-start gap-2 rounded-lg border p-3 my-3 text-left text-sm transition-all hover:bg-accent max-w-[600px]',
+                    chunkId === it.id && 'bg-accent',
+                  )}>
+                  <div className="flex w-full flex-row justify-between items-center mb-2">
+                    <p className="text-sm font-bold">{it.id}</p>
+                    <p className="text-xs font-normal">
+                      {formatDistanceToNow(new Date(it.createdAt), {
+                        addSuffix: true,
+                      })}
+                    </p>
+                  </div>
+                  <p className="text-xs text-muted-foreground">
+                    {format(
+                      new Date(it.updatedAt),
+                      "eeee, MMMM do, yyyy 'at' hh:mm a",
+                    )}
+                  </p>
+                  <p className="text-xs text-slate-400 font-medium font-mono">
+                    {it.projectId}
+                  </p>
+                  <div className="bg-slate-50 p-2 rounded-lg">
+                    <p className="text-wrap line-clamp-3 text-[10px] text-slate-400 font-mono font-extralight break-all">
+                      {it.data}
+                    </p>
+                  </div>
+                </Link>
```

Comment:
moved to it's abstracted component

Reply:
good idea 👍

Reply:
🤔  also, what do you think about moving the format(`"eeee, MMMM do, yyyy 'at' hh:mm a"`) into a constant? it repeats in several places

Reply:
updated